src/main/java/Application.java

```diff
@@ -0,0 +1,74 @@
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import static spark.Spark.get;
+import static spark.Spark.post;
+
+
+public class Application {
+
+    private static final Logger logger = LogManager.getLogger("Application");
+
+    public static void main(String[] args) {
+        logger.info("Starting spin text API");
+        logger.info("Initializing dictionary");
+        Dictionary dict = new Dictionary();
+        try {
+            dict.initialize();
+            logger.info("Initialization successful");
+        } catch (Exception e){
+            logger.error("Initialization failed");
+            logger.error(e);
+        }
+
+        get("/hello", (request, response) -> {
+            logger.info("New request received");
+            return "world";
+        });
+        post("/sg", (request, response) -> {
+            Session session = new Session(dict.getWord());
+            int gameID = session.startGame();
+            System.out.println("Mystery word is : "+session.getMysteryWord());
+            //response.redirect("/sg/game/" + gameid);
+            return gameID;
+        });
+
+        get("/sg/:id", (request, response) -> {
+            int id = Integer.parseInt(request.params(":id"));
+            Session ss = Session.getGame(id);
```

Comment:
maintain the sessions elsewhere. Not in Session class. It is violation of Single Responsible principle.